kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -91,7 +91,7 @@
               class="login-btn"
               :text="$tr('nextLabel')"
               :primary="true"
-              :disabled="busy"
+              :disabled="!isNextButtonEnabled"
```

Comment:
You'll need to put a guard in the `signIn` method as well to prevent a submit event on the kwyboard. Even though the "next" button is disabled, I can still press "Enter" on my keyboard and move forward with an invalid username like `jb!!`